Packs/GoogleChatViaWebhook/Integrations/GoogleChatViaWebhook/GoogleChatViaWebhook.py

```diff
@@ -121,17 +121,10 @@ def send_google_chat_message_command(client: Client, message: str, threadName: O
     """
     res = client.send_google_chat_message(message=message, threadName=threadName)
     result = {
+        'Name': res.get('name'),
         'Message': res.get('text'),
-        'SpaceName': res.get('space').get('name'),
-        'SpaceDisplayName': res.get('space').get('displayName'),
-        'SpaceType': res.get('space').get('type'),
-        'CreatedTime': res.get('createTime'),
-        'ThreadReply': res.get('threadReply', False),
         'ThreadName': res.get('thread').get('name'),
-        'Name': res.get('name'),
-        'SenderDisplayName': res.get('sender').get('displayName'),
-        'SenderName': res.get('sender').get('name'),
-        'SenderType': res.get('sender').get('type')
+        'Space': res.get('space').get('name')
```

Comment:
The previous version called it 'SpaceName' lets leave it the same

Reply:
Done